### ISSUE TITLE
Confirmation of deletion of bookmarks

### ISSUE THREAD

**Issue by don-moelb** [mapsme/omim#12903]
Hi
First of all. I love MapsMe. I use it all the time on my travels, business and private. I used it extensively on my 3 months trip through China where Google maps is of no use at all. This is also where the problem starts.
I found that I made 2 separate bookmark lists of that travel. When I copied the bookmarks from one group to the other I wanted to sort them by name but accidentally hit the button delete list. The list with all my bookmarks was gone. That's it. Disapeared into thin air. No questions asked.
I'm going to send my phone to a professional file rescue center, pay 350€ for the small chance to get back the file with the booksmarks. And actually I think this is quite annoying.

So here is my suggestions, and I really hope you consider implementing it in an upcoming release:
Please implement a Confirmation for the deletion of bookmark lists in your app. So if one presses the delete button a message pops up and asks if you really want to delete it. You can even make a second confirmation that tells the user, that a deletion is irreversible and files can't be restored after deletion.

By the way: If the file recovery service would find the KMB file is there a way I can import it into MapsMe? I just found that this is possible with KMZ and KML files. But with KMB files?
If this is not possible i can spare the recovery service if I can't use the KMB file afterwards.

**Comment by mpimenov** [comment]
> If the file recovery service would find the KMB file is there a way I can import it into MapsMe?

Yes, it's possible, though probably not from within the app. Contact our support team and/or ping this issue if you restore your KMB.

Am I right that you were not using our bookmarks-in-the-cloud option?

**Comment by don-moelb** [comment]
No I didn't use the bookmarks in the cloud option. Recently I worked out a new back up plan for my computers and devices, PC, Phone. So I have all data from my phone backed up: Photos, Documents, etc., etc. In this whole scheme I didn't think of the bookmarks in MapsMe and how easily you can delete them by accident.

**Comment by thany** [comment]
> You can even make a second confirmation that tells the user, that a deletion is irreversible and files can't be restored after deletion.

No, make it undoable. That's a much, MUCH better experience. Undo can be performed at a later stage. A confirm box is only there at one unfortunate moment.
